### ISSUE TITLE
Redirecting all traffic to https

### ISSUE THREAD

**Issue by geoffwalmsley** [AzuraCast/AzuraCast#4115]
**Using Docker installation method**
Yes - DO App image

**AzuraCast version**
Stable 

**Host Operating System**
Ubuntu 20.04

**Describe the bug**
When I visit my domain xchradio.com, it goes in as a port 80 request, so shows an insecure connection. If I type in the https:// before the domain, I get a nice secure connection. 

I have tried:
- overriding docker-compose.yml, to add a redirect for all port 80 traffic to 443 (failed, with many connection errors in web log)
- force_renew ssl cert
- Tried to create another docker container to host a static site and just embed the player, but ran into further issues with that.

It would be really great to know if there is a proper way to do this within the azuracast structure. My guess is that I should be overriding docker-compose, but I'm struggling to connect how I would write an override conf and then where it should be applied. I have read the other issues about https and can see it has been made a lot smoother to get working. I figure there is something relatively simple that I'm just not getting

I am really loving this software so I'd love to get it working perfectly and properly :)

**Comment by SC2Mitch** [comment]
Have you enabled the `Always Use HTTPS` in your system settings?

**Comment by SlvrEagle23** [comment]
@geoffwalmsley If you visit `Administration` > `System Settings` with a working secure connection, you can enable the "Always Use HTTPS" setting to force HTTPS on all connections. We limit this setting to only be controllable from a secure connection to ensure you have one set up before enabling it, as we've had many incidents of users not having set up their SSL certificates before enabling the setting.

**Comment by geoffwalmsley** [comment]
Hey, thanks so much for your reply! 

What was happening (and I can't believe i didn't mention it last night) is that the Always use HTTPS checkbox was disabled. I read in a comment in another issue that you'd changed the nginx proxy to ensure that the stream would always be SSL. So I just assumed that button was intentionally disabled because it had been changed behind the scenes and not removed from the front end for some reason. Anyway, I just tried manually editing the page html and forcing it to check Always use HTTPS. And now everything works fine! Could just be a quirk for me on chrome, and it doesn't seem like anyone else has had the issue before me.

BTW - I like your video about all the product features and stuff - it would be cool to see more like it. Video tutorials for common setup configurations, how to get the best out of the software, tips and tricks type stuff. The installation process and docs are amazing too. I love azuracast :)
